src/makeInfinityAdder.js

```diff
@@ -4,7 +4,25 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let result = 0;
+
+  const infinityAdder = (firstNumber) => {
+    const saveResult = [];
```

Comment:
The `saveResult` array is unnecessary. You can simply return the `result` and reset it afterwards without using an array.